tests/Fixtures/TestBundle/Entity/Foo.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Foo.
+ *
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ *
+ * @ApiResource(attributes={
+ *     "order"={"name", "DESC"}
```

Comment:
IMO it's useful (ordering by name then firstname is very common for instance). You're right for the syntax, my example is faulty (missing `order` key). A PR would be great!